### PR TITLE
Add GPU support Kolmogorov Solver

### PR DESCRIPTION
**Pull request by ashutosh-b-b**

### CODE REVIEW

**Comment by ashutosh-b-b** [comment]
@ChrisRackauckas This should provide a GPU support.

File: src/kolmogorov_solve.jl [hunk 1]
```diff
@@ -60,7 +61,14 @@ function DiffEqBase.solve(
         x_sde = hcat(x_sde , u)
     end
     y = phi(x_sde)
+    if use_gpu == true
+        y = y |>gpu
+        xi = xi |> gpu
```
Comment (ChrisRackauckas) [inline src/kolmogorov_solve.jl:66]: Our convention is to match the user intention, so instead adapt it to match `u0`

Comment (ashutosh-b-b) [reply src/kolmogorov_solve.jl:66]: But `u0` will either be a distribution  or there exists no `u0` for KolmogorovPDEProblem?

Comment (ChrisRackauckas) [reply src/kolmogorov_solve.jl:66]: what's the user data that's an array?

Comment (ashutosh-b-b) [reply src/kolmogorov_solve.jl:66]: There's no array input. Two tuples though. `xspan` and `tspan`

Comment (ashutosh-b-b) [reply src/kolmogorov_solve.jl:66]: I thought the only way to know is to take a boolean itself to check if the user wants to use a GPU

Comment (ChrisRackauckas) [reply src/kolmogorov_solve.jl:66]: I see, this is fine then.

**Comment by ChrisRackauckas** [comment]
@christopher-dG could we get GPU CI setup on this repo? It happened sooner than I thought it would 😆

**Comment by christopher-dG** [comment]
Which test group should be run?

**Comment by ChrisRackauckas** [comment]
Make a new one for GPU. We can add stuff into there.

**Pull request merged**